third_party/cpgfunctionEP-0.3.2/CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+cmake_minimum_required(VERSION 3.6)
+project(cpgfunctionEP)
+
+set(CMAKE_CXX_STANDARD 17)
+if(CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} --std=c++17")
```

Comment:
`set(CMAKE_CXX_STANDARD 17)` will take care of adding `-std=c++17`, and here you have an extra `-` anyways too.

Reply:
Believed to be addressed in a7f66c005851db3116f729a4ad814d4d86fd0d41